src/assets/articleAssets/demo2.scss

```diff
@@ -499,3 +499,42 @@ a:hover {
 }  
   
   
+
```

Comment:
Separate the tag styles in a different file, and rename `demo2.scss` to a more descriptive file name